packages/common-ui-web/README.md

```diff
@@ -0,0 +1,40 @@
+# @monkvision/common-ui-web
+This is a collection of reusable and customizable UI components for React web applications. It is designed to help developers easily build modern and responsive user interfaces.
+
+# Installing
+To install the package, you can run the following command :
+
+```shell
+yarn add @monkvision/common-ui-web
+```
+
+# Basic Usage
+Import and use the components in your project:
+
+```typescript jsx
+import { SVGElement } from '@monkvision/common-ui-web';
+
+function App() {
+  return (
+    <div>
+      <SVGElement ... />
+      <SightOverlay ... />
+    </div>
+  );
+}
+```
+
```

Comment:
Before `## SVGElement` please add a subsection header like this :

`# Available components`

Reply:
Done